plugins/anghami/integration.js

```diff
@@ -0,0 +1,66 @@
+var playbackStatus = mellowplayer.PlaybackStatus.STOPPED;
+
+function update() {
+    return {
+        "playbackStatus": playbackStatus,
+        "canSeek": false,
+        "canGoNext": true,
+        "canGoPrevious": true,
+        "canAddToFavorites": false,
+        "volume": 1,
+        "duration": 0,
+        "position": 0,
+        "songId": 0,
```

Comment:
songId must be set. You can use the below method to generate a unique id based on the song title:

```javascript
function getHashCode(s) {
    return s.split("").reduce(function(a, b) {
        a = ((a << 5) - a) + b.charCodeAt(0);
        return a & a
    }, 0);
}
```

Reply:
I couldn't figure out what songID is just by looking at the documentation, can you please make it more clear like this comment? thanks.

Reply:
You're right, the documentation is not clear. I've updated it. Thank you for the feedback!

Reply:
@ColinDuquesnoy You're welcome.